.ci.yaml

```diff
@@ -292,21 +292,21 @@ targets:
     timeout: 30
     properties:
       target_file: analyze_legacy.yaml
-      channel: "3.22.0"
+      channel: "3.22.3"
```

Comment:
This change looks wrong, but that's only because I apparently messed this up last time. It should have been 3.19.0 prior to this PR, which means we've been analyzing with N-2, but not actually with N-1, for the past stable cycle 😬